src/rest/container.rs

```diff
@@ -29,6 +29,7 @@ impl Container {
     pub fn authorise(client        : ::std::sync::Arc<::std::sync::Mutex<::safe_client::client::Client>>,
                      container_info: Option<::rest::ContainerInfo>) -> Result<Container, ::errors::NfsError> {
         let directory_helper = ::helper::directory_helper::DirectoryHelper::new(client.clone());
+        debug!("Retrieving directory matching container info ...");
```

Comment:
Remove. Instead, in the branching put statements:
`Authorising user root-container ...`
and
`Authorising specific container ...`
according to if in `Some()` or `None` branch.